packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -129,6 +129,7 @@ const builtInFlags = [
         type: Boolean,
         negative: true,
         description: 'Enables Hot Module Replacement',
```

Comment:
`Enables/Disables Hot Module Replacement`, we are using it for any boolean flags with negative in docs

Reply:
Currently, this is how the description shows up for built-in negated flags:-

<img width="531" alt="Screenshot 2020-12-04 at 7 13 24 PM" src="https://user-images.githubusercontent.com/25279263/101170677-cda98300-3664-11eb-9772-ecca4f3fcf89.png">

Reply:
oh, yep, right

Reply:
Anyway we should solve this for all negative flag

Reply:
Currently, this isn't possible for core flags since we aren't receiving enough information.

Reply:
What is not enouth?

Reply:
We already have the built-in flags as part of [cli-flags](https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/utils/cli-flags.js), and hence adding a key corresponding to the description for negated flags would do. However, this isn't the case with core flags; the description for a negated flag is to be made available.

Reply:
We already have the built-in flags as part of [cli-flags](https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/utils/cli-flags.js), and hence adding a key corresponding to the description for negated flags would do. However, this isn't the case with core flags; the description for a negated flag is to be made available.

Reply:
What we output for `--no-amd`?

Reply:
Negates `amd`

Reply:
Can you provide example? For help output

Reply:
The description for built-in flags show up as provided, while core flags go by the format `Negates ${flag.name}`.

<img width="636" alt="Screenshot 2020-12-04 at 6 12 38 PM" src="https://user-images.githubusercontent.com/25279263/101169776-75be4c80-3663-11eb-842c-7fb2576ae140.png">